build.py

```diff
@@ -249,19 +257,24 @@ def releaseId() :
 
 # Perform the build.
 
+devTools = ( 'source scl_source enable %s && ' % args.devTools ) if args.devTools else ""
+
 if args.project == "gaffer" :
 
 	# We run SCons indirectly via `python` so that it uses our
 	# preferred python from the environment. SCons itself
 	# unfortunately hardcodes `/usr/bin/python`, which might not
 	# have the modules we need to build the docs.
-	buildCommand = "python `which scons` package PACKAGE_FILE={uploadFile} ENV_VARS_TO_IMPORT=PATH DELIGHT_ROOT={delight} ARNOLD_ROOT={arnoldRoot} OPTIONS='' -j {cpus}".format(
-		cpus=multiprocessing.cpu_count(), **formatVariables
+	buildCommand = "{devTools} python `which scons` package PACKAGE_FILE={uploadFile} ENV_VARS_TO_IMPORT=PATH DELIGHT_ROOT={delight} ARNOLD_ROOT={arnoldRoot} OPTIONS='' -j {cpus}".format(
```

Comment:
Just noting the conclusion from our recent chat : this will break the mac build, and we'd be better to put the `scl enable` stuff up above with the `--entrypoint` stuff in the `containerCommand`.